openmc/mgxs/library.py

```diff
@@ -239,22 +255,34 @@ def name(self, name):
 
     @mgxs_types.setter
     def mgxs_types(self, mgxs_types):
+        all_mgxs_types = np.append(openmc.mgxs.MGXS_TYPES,
+                                   openmc.mgxs.MDGXS_TYPES)
```

Comment:
Is there a reason you want an `ndarray` here rather than a list, i.e. `openmc.mgxs.MGXS_TYPES + openmc.mgxs.MDGXS_TYPES`?